Ghpr.CouchDb/CouchDbDatabase.cs

```diff
@@ -0,0 +1,63 @@
+﻿using Ghpr.CouchDb.Entities;
+
+namespace Ghpr.CouchDb
+{
+    public class CouchDbDatabase
```

Comment:
Thei layer of abstraction does nothing. You can use underlying CouchDbClient instead

Reply:
Anyway better approach is to inject client as readoly private property via constuctor rather then wrap each call with using